connector/src/main/java/org/geysermc/connector/network/translators/item/ItemTranslator.java

```diff
@@ -126,12 +122,20 @@ public static ItemData translateToBedrock(GeyserSession session, ItemStack stack
 
         ItemEntry bedrockItem = ItemRegistry.getItem(stack);
 
-        ItemStack itemStack = new ItemStack(stack.getId(), stack.getAmount(), stack.getNbt() != null ? stack.getNbt().clone() : null);
+        com.github.steveice10.opennbt.tag.builtin.CompoundTag nbt = stack.getNbt() != null ? stack.getNbt().clone() : null;
```

Comment:
Does cloning the NBT mess with the process? Since the individual translators modify the NBT reference.

Reply:
The NBT data was cloned before I just moved it.

Reply:
Oops, I see.